src/presenter/trip-points-presenter.js

```diff
@@ -23,9 +24,12 @@ export default class TripPointsPresenter {
     this.#offers = this.#pointsModel.offers;
     this.#destinations = this.#pointsModel.destinations;
 
-    render(this.#tripList, this.#eventContainer);
-
-    this.#tripPoints.forEach((tripPoint) => this.#renderTripPoint(tripPoint, this.#offers, this.#destinations));
+    if (this.#tripPoints.length === 0) {
+      render(new EmptyPointView(), this.#eventContainer);
+    } else {
```

Comment:
Предлагаю без else ветки обходится когда можно, написав guard expression 
```suggestion
    if () {
      ....
      return
    }
    
    .....
```